### DISCUSSION TITLE
Copilot not translate to russia

### DISCUSSION THREAD

**Discussion by abdurashid-dev** [community/community#14934] [Copilot Conversations]
i work with laravel framework when i want to add another language copilot helps me but when i decide translate to russia copilot stop to help me

**Comment by Kek5chen** [comment]
![image](https://user-images.githubusercontent.com/52585984/163473558-0299fe1f-20e0-443e-9a22-4bba463f6157.png)
;) Try giving it more context.

**Reply by Kek5chen** [reply]
It returns it because it wasnt sure how to print something as i did this in a framework which did not explicitly work that way. Hope it helps either way!
